### PR TITLE
Update AUTHORS

### PR DESCRIPTION
**Pull request by certik**

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -62,4 +62,6 @@ Kieran Kaempen <kaempen2@illinois.edu>
 Clouds Flowing <clouds.flowing@gmail.com>
 Sylvain Corlay <sylvain.corlay@gmail.com>
 Alan Hu <alanlh2@illinois.edu>
-
+Rajiv Ranjan Singh <rajivperfect007@gmail.com>
+Francesco Bonazzi <franz.bonazzi@gmail.com>
+Simon Stelter <stelter@tzi.de>
```
Comment (isuruf) [inline AUTHORS]: @ichumuh, @Upabjojr, @iamrajiv, let us know if you want to change your name or email here.

Comment (ichumuh) [reply AUTHORS]: wow thanks for that! I'd prefer simon.stelter@gmail.com as my email.

**Comment by Upabjojr** [comment]
I'm ok with that. Thanks.

**Comment by asmeurer** [comment]
You might want to copy the authors/mailmap scripts from SymPy (at any rate, the SymPy script uses `--topo-order`, which I think is more correct).

**Comment by asmeurer** [comment]
We could also refactor those scripts into a separate project, so that anyone can use them.

**Comment by certik** [comment]
I've added the `--topo-order`, the authors got slightly rearranged, but no big deal I think.

We can use the SymPy machinery in another PR. I was hoping to avoid it, as this script seems simple enough, we do not need a custom rearrangement.

**Comment by asmeurer** [comment]
IIRC --topo-order correctly handles the case where you have very old commits that are merged in later, for example, a PR that isn't merged for a while. You want the author to be ordered based on when the PR is merged, not when they made their commit.

The SymPy tooling is also nice for .mailmap. Maybe I'll split them out as separate tools myself some day.

**Comment by certik** [comment]
Yes, I agree `--topo-order` is the order we want.

**Pull request merged**